frontend/src/core/loaders/components/WaveLoader.css

```diff
@@ -0,0 +1,101 @@
+
+.overlay {
+    background: #3F4752;
+    display: table;
+    height: 100%;
+    overflow: hidden;
+    position: fixed;
+    width: 100%;
+    z-index: 14;
+}
+
+/*
+* SpinKit
+* https://github.com/tobiasahlin/SpinKit/blob/master/LICENSE
+*/
+
+#project-load {
+    background: #3f4752;
+    margin-top: 60px;
+}
+
+#project-load .animation {
+    margin: 20px auto;
+    width: 100px;
+    height: 60px;
+    text-align: center;
+    font-size: 20px;
+}
+
+#project-load .text {
+    font-size: 1.5em;
+    opacity: 0;
+    text-align: center;
+}
+
+#project-load .animation > div {
+    background-color: #7BC876;
+    height: 100%;
+    width: 12px;
+    display: inline-block;
+
+    -webkit-animation: stretchdelay 1.2s infinite ease-in-out;
+    animation: stretchdelay 1.2s infinite ease-in-out;
+}
+
+#project-load .animation div:nth-child(2) {
+    -webkit-animation-delay: -1.1s;
+    animation-delay: -1.1s;
+}
+
+#project-load .animation div:nth-child(3) {
+    -webkit-animation-delay: -1.0s;
+    animation-delay: -1.0s;
+}
+
+#project-load .animation div:nth-child(4) {
+    -webkit-animation-delay: -0.9s;
+    animation-delay: -0.9s;
+}
+
+#project-load .animation div:nth-child(5) {
+    -webkit-animation-delay: -0.8s;
+    animation-delay: -0.8s;
+}
+
+@-webkit-keyframes stretchdelay {
+    0%, 40%, 100% { -webkit-transform: scaleY(0.4) }
+    20% { -webkit-transform: scaleY(1.0) }
+}
+
+@keyframes stretchdelay {
+    0%, 40%, 100% {
+        transform: scaleY(0.4);
+        -webkit-transform: scaleY(0.4);
+    }
+    20% {
+       transform: scaleY(1.0);
+       -webkit-transform: scaleY(1.0);
+   }
+}
+
+/*
+* Loading text
+ */
+#project-load .text {
+    display: block;
+    animation: fadeInLoaderText 1s forwards;
+    animation-delay: 3s;
+}
+
+@keyframes fadeInLoaderText {
+    0% {
+        display: None;
+        opacity: 0;
+    }
+
+    100% {
+        display: block;
+        opacity: 1;
+    }
+}
```

Comment:
You should configure your code editor to always add new lines at the end of files. And add a new line here. ;)